Corona-Warn-App/src/main/res/values/styles.xml

```diff
@@ -223,6 +223,9 @@
     <style name="headline6" parent="@style/TextAppearance.MaterialComponents.Headline6">
         <item name="android:textColor">@color/colorTextPrimary1</item>
     </style>
+    <style name="headline7" parent="@style/TextAppearance.MaterialComponents.Headline5">
```

Comment:
:) headline7 = headline5 + bold ?

Reply:
Exactly :) There wasn't a headline with 24sp and bold defined up until now. If there's a better way of doing this, I'm happy to implement it.

Reply:
working :-) but wouldn't it be better to have a proper stylenaming, e.g. headline5-normal or headline5-bold..
headline1234567 is a little bit odd

Reply:
Yeah, that makes sense. I'll change it.